practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -67,7 +68,12 @@ const FormValidation = (props) => {
       setErrors(newErrors);
 
       if (Object.keys(newErrors).length === 0) {
+        const createCustomer = createCustomerService(formData);
+        handleCreateCustomer(createCustomer);
         onShowForm();
+      } else {
+        setIsCreateCustomerFailed(true);
+        return;
```

Comment:
why need return in here

Reply:
That is a handle error for creating customer failed that I haven't done yet